### ISSUE TITLE
Faction database

### ISSUE THREAD

**Issue by elstringer** [doomsider/shadow#8]
hello is it possible to have the faction in a table in the database ?

**Comment by doomsider** [comment]
Yes this is planned but I have to rewrite a good section of the database and how it works.
